chainforge/react-server/src/backend/backend.ts

```diff
@@ -671,7 +671,7 @@ export async function queryLLM(id: string,
       console.log(`Querying ${llm_str}...`)
 
       // Yield responses for 'llm' for each prompt generated from the root template 'prompt' and template variables in 'properties':
-      for await (const response of prompter.gen_responses(_vars, llm_str as LLM, num_generations, temperature, llm_params, chat_hists)) {
+      for await (const response of prompter.gen_responses(id, _vars, llm_str as LLM, num_generations, temperature, llm_params, chat_hists)) {
         
         // Check for selective failure
         if (response instanceof LLMResponseError) {  // The request failed
```

Comment:
Right around here is where you'd check if `QueryStopper` has the id, then you'd throw an error. In line 697 below, you'd bubble that error up the chain, but you'd use `instanceof` your custom Error class, instead of a string check.